### DISCUSSION TITLE
send notification by email to end user

### DISCUSSION THREAD

**Discussion by m-rjoob** [snipe/snipe-it#16002] [Ideas]
I’d like to suggest a new feature:  

When the IT admin makes changes in the system, such as assigning a new asset to an end user, can the system automatically send an email notification to the targeted user? This notification would inform them that IT has added a new asset to their account and include details about the asset.  

Does this make sense?

**Comment by swift2512** [comment]
This may be helpful
![image](https://github.com/user-attachments/assets/e6f619b2-1fd5-4882-8ac0-d932b4391da5)

**Comment by m-rjoob** [comment]
@swift2512 is there any additional setting i need to check because i checked this option and once i try to check in/out items under this category i get the attachment error 
![Screenshot 2025-01-16 at 8 50 11 AM](https://github.com/user-attachments/assets/a8775124-56df-4cdf-848c-614dd13fa04a)

**Reply by swift2512** [reply]
Mail server details may be missing in **.env** file.
![image](https://github.com/user-attachments/assets/0591da8e-a5f7-472a-84b9-7c3ff9ee3508)
